DatadogSDKObjc.podspec

```diff
@@ -22,4 +22,5 @@ Pod::Spec.new do |s|
 
   s.source_files = "Sources/DatadogObjc/**/*.swift"
   s.dependency 'DatadogSDK', s.version.to_s
+  s.dependency 'DatadogLogs', s.version.to_s
```

Comment:
ObjC target won't be modularised or it's temporary?

Reply:
Indeed, the objc interface will be exposed in a single module for now. Ultimately, we should find a way to include the objc interface in each module, without polluting swift completion.